src/db/query.js

```diff
@@ -0,0 +1,15 @@
+import pool from "./connection";
+
+const text = "INSERT INTO meetups(location,images,topic,happeningOn,tags) VALUES($1,$2,$3,$4,$5)";
```

Comment:
please use descriptive and meaningful names for your variables. text does not provide context as to what this line of code is doing. please rename to a more meaningful and suitable name

Reply:
ok this below text  just a query to test my database setup I have refactored it in my query 
```suggestion
const text = "INSERT INTO meetups(location,images,topic,happeningOn,tags) VALUES($1,$2,$3,$4,$5)";
```